packages/@angular/cli/blueprints/ng/files/package.json

```diff
@@ -12,21 +12,21 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/compiler": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/core": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/forms": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/http": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/platform-browser": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/platform-browser-dynamic": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/router": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^3.4.0' %>",
+    "@angular/common": ">=4.0.0-beta <5.0.0",
+    "@angular/compiler": ">=4.0.0-beta <5.0.0",
+    "@angular/core": ">=4.0.0-beta <5.0.0",
+    "@angular/forms": ">=4.0.0-beta <5.0.0",
+    "@angular/http": ">=4.0.0-beta <5.0.0",
+    "@angular/platform-browser": ">=4.0.0-beta <5.0.0",
+    "@angular/platform-browser-dynamic": ">=4.0.0-beta <5.0.0",
+    "@angular/router": ">=4.0.0-beta <5.0.0",
```

Comment:
I actually left these there while waiting for 4.0.0 to be released, have to update it.